tests/e2e/tests/pilot/util/config.go

```diff
@@ -29,7 +29,8 @@ const (
 
 // Config defines the configuration for the test environment.
 type Config struct {
-	KubeConfig            string
+	KubeConfig            [2]string
```

Comment:
Consider adding another field like RemoteKubeConfig instead. A lot of changes (such as adding '[0]' to refer to the local cluster) can be avoided. 
And the code can be structured according to that. For example, deployApp, deployAppRemote().

Reply:
I was on the fence about doing that.   The reason I created as a array is that we might want to have more than just 2 remote clusters in the future.   I wasn't sure how easy it was to create more than two clusters in the CI system though.   I will also get Tim's opinion on the change.